addons/website_livechat/views/website_livechat.xml

```diff
@@ -2,13 +2,157 @@
 <openerp>
     <data>
 
-<template id="header" inherit_id="website.layout" name="LiveChat Import (internal) Scripts">
-    <xpath expr="//body" position="inside">
-        <t t-if="website and website.channel_id">
-            <t t-raw="website.channel_id.script_internal"/>
-        </t>
-    </xpath>
-</template>
+        <template id="header" inherit_id="website.layout" name="LiveChat Import (internal) Scripts">
+            <xpath expr="//body" position="inside">
+                <t t-if="website and website.channel_id">
+                    <t t-raw="website.channel_id.script_internal"/>
+                </t>
+            </xpath>
+        </template>
+
+
+
+        <!-- Layout add footer -->
+        <template id="header_footer_custom" inherit_id="website.footer_default" name="Footer Rating Livechat Link">
+            <xpath expr="//ul[@id='products']" position="inside">
+                <li><a href="/livechat/">Livechat Support</a></li>
+            </xpath>
+        </template>
+
+        <!-- Page Layout -->
+        <template id="channel_page" name="Livechat Channel Satisfaction Page" page="True">
+            <t t-call="website.layout">
+              <div id="wrap">
+                <div class="container">
+                    <h1><span>Livechat Channel</span>  <small t-field="channel.name" /></h1>
+
+                    <div t-field="channel.website_description" class="oe_structure mt16" />
+
+                    <div class="row mt32">
+                        <div class="col-md-8">
+                            <t t-if="len(ratings) &gt; 0">
+                                <div class="row">
+                                    <div class="col-md-12 mb32">
+                                        <h3>Statistics</h3>
+                                        <div class="row">
```

Comment:
Is it necessary to have row -> col-md-12 -> row ? Can't this layer be removed to simplify the layout ?